tasks/set_vars.yml

```diff
@@ -1,11 +1,11 @@
 ---
 # tasks/set_vars.yml
 
-- name: Include customer env_type vars
+- name: Include global customer & env_type vars
   include_vars:
     file: "{{ item }}"
   with_items:
-    - "group_vars/customer/{{ customer }}.yml"
+    - "group_vars/customer/{{ customer }}/main.yml"
```

Comment:
Why not all files in customer dir ?

Reply:
I think we need to control the importation sequence, WDYT?